README.md

```diff
@@ -94,7 +94,9 @@ Return values from the command-line validator are as follows:
 * `0 (Success)`.  There were no validation (or any other) errors.
 * `1 (Validation Error)`.	All specified files were validated but at least one failed validation.
 * `2 (Usage Error)`.  There was a problem interpreting command-line arguments.  No validation was performed.
-* `3 (File Error)`.  There was a problem reading one or more files or directories.  Validation may have been performed on a subset of specified KBs.  If there is an error loading any ontologies or SHACL files, then no validation is performed.
+* `3 (File Error)`.  A file was rejected, either due to an I/O error or because it didn't meet certain criteria.
```

Comment:
Should we make these codes bit values (0, 1, 2, 4)? We could then return an error code that combines them and wouldn't have 1(Validation Error) clobber 3(File Error). What do you think?

Reply:
We could but...that seems maybe a little unnecessarily complex.  It's true that Validation Error clobbers File Error, but the thinking was that it's more important and more common.  How about if I leave it as is but document this behavior here in the README?

Reply:
That's fair. The output also gives a good indication as to what's going on. I'm OK even if you don't want to update the README. I'll leave it up to you.